hello/patch/README.md

```diff
@@ -0,0 +1,9 @@
+# Patch Workflow Sample
+This sample will demonstrate how to version a workflow. The sample workflow will execute a activity and fire a timer. 
```

Comment:
As all of the other READMEs do, make sure to point people to the primary README for prerequisites

Reply:
N/A